style.css

```diff
@@ -51,27 +53,177 @@
   line-height: 24px;
   color: #344563;
   margin-top: 12px;
-  margin-left: 24px;
-  margin-right: 24px;
 }
 
 .details .connect {
   display: flex;
   font-weight: 500;
   font-size: 16px;
+  margin-top: 12px;
   line-height: 24px;
   color: #7f8cff;
-  margin-left: 24px;
-  margin-top: 12px;
 }
 
 .social-links {
   display: flex;
-  margin-left: 14px;
   margin-top: 12px;
 }
 
 .social-icons {
   color: #505f79;
   padding: 20px 12px;
 }
+
+/* ---------------------------works------------------------------------------ */
+
+.shadow-back {
+  background: #e5e5e5;
+  padding: 114px 24px;
+  display: grid;
+  flex-direction: column;
+  margin-top: 114px;
+  justify-content: center;
+}
+
+.works {
+  padding: 16px;
+  margin-top: 114px;
+  background: #fff;
+  width: 327px;
+  height: 552px;
```

Comment:
- [ ] As per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma), let's stick to the design by giving a proper height to the card content and simply making it `width:auto` to make it depend on its content.

- Your desig
![workcard](https://user-images.githubusercontent.com/97834160/205104184-6b517a82-ae3a-47e7-ab7c-b3a97b370a67.png)
n:

- Figma design:
![figmaWorkCARD](https://user-images.githubusercontent.com/97834160/205104593-b8da1607-7e1a-4ffd-acc1-e44b4222783f.png)